### PR TITLE
add Pavan's hackbar

### PR DESCRIPTION
**Pull request by pavan-areti**
Did all steps as mentioned

### CODE REVIEW

File: _hackbars/pavan.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+author: pavan
+---
+
+In open source, we feel strongly that to really do something well, you have to get a lot of people involved.
+
+---
+
+## We have only one earth, save it! ⏱
+
+![Ending Earth]({{site.baseurl}}/assets/images/pavan.jpg)
```
Comment (tasnimzotder) [inline _hackbars/pavan.md]: hey @pavan-areti, got an extension error -

```suggestion
![Ending Earth]({{site.baseurl}}/assets/images/pavan.jpeg)
```

**Comment by pavan-areti** [comment]
i changed it ,please check

**Comment by tasnimzotder** [comment]
Okay

**Review by tasnimzotder — APPROVED** [review]

**Comment by tasnimzotder** [comment]
Thanks for your contributions :)

**Pull request merged**